solr/core/src/java/org/apache/solr/handler/SnapShooter.java

```diff
@@ -277,7 +281,8 @@ public void createSnapAsync(final int numberToKeep, Consumer<NamedList<?>> resul
    * @see IndexDeletionPolicyWrapper#saveCommitPoint
    * @see IndexDeletionPolicyWrapper#releaseCommitPoint
    */
-  protected NamedList<Object> createSnapshot(final IndexCommit indexCommit) throws Exception {
+  protected SnapShooterBackupCoreResponse createSnapshot(final IndexCommit indexCommit)
```

Comment:
[Q] Looking at createSnapAsync, it looks like this same `createSnapshot()` code is also called by Solr's "ReplicationHandler" (i.e. `/replication`) APIs.

Maybe the name of the response class should be made more generic so that it's not so "Backup Core API" specific?  Maybe something like `CoreSnapshotResponse` or `CoreSnapshotDetails`?  Idk, just thinking aloud...

Reply:
`CoreSnapshotResponse` looks good because most of our JerseyResponse classes ends with 'Response' keyword. There is another class named `IncrementalBackupCoreResponse.java`. Should I change it to `IncrementalShardSnapshotResponse.java`.

Reply:
Sure, renaming them both consistently makes sense to me?